src/Controls/src/Core/HybridWebView/HybridWebView.cs

```diff
@@ -183,13 +106,7 @@ private void AsyncTaskCompleted(string taskId, string result)
 				return default;
 			}
 			return JsonSerializer.Deserialize<TReturnType?>(stringResult, returnTypeJsonTypeInfo);
-	}
-#else
-		public Task<TReturnType?> InvokeJavaScriptAsync<TReturnType>(string methodName, object returnTypeJsonTypeInfo, object?[]? paramValues, object?[]? paramJsonTypeInfos)
-		{
-			throw new NotImplementedException();
 		}
-#endif
 
 		/// <inheritdoc/>
 		public async Task<string?> EvaluateJavaScriptAsync(string script)
```

Comment:
This method still does all the work inside the xplat view so maybe it also needs the same treatment as the `InvokeJavaScriptAsync`? Also, for linker paths, maybe we should be using the `OperatingSystem.IsPlatformName()` so it can be linked out correctly instead of hitting the OS check API at runtime.